olm-catalog/serverless-operator/project.yaml

```diff
@@ -22,13 +22,14 @@ requirements:
     label: 'v4.6'
 
 dependencies:
-  serving: 1.1.2
-  kourier: 1.1.0
-  net_istio: 1.1.0
+  serving: 1.2.0
+  kourier: 1.2.0
+  net_istio: 1.2.0
   maistra: 2.1.0
 
   eventing: 1.2.1
   eventing_kafka: 1.1.0
   eventing_kafka_broker: 1.1.3
+
   cli: 1.1.0
-  operator: 1.0.1
+  operator: 1.2.0
```

Comment:
We need to update the operator to address the issue with the limits applying to all containers across containers.

Reply:
This slightly touches the eventing crd too @matzew could you check pls?

Reply:
Would it be possible to do that bump on a separate PR ? 
Or is there a hard dependency that it must be done in this "serving and friends" bump?

Reply:
Ok I can try that.

Reply:
No, but not sure why the operator is also bumped, when we bump serving (PR says: serving bump).

I personally like these to be all small, and less or more atomic, instead of a huge PR that bumps `n` components, and errors are harder to isolate